src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHetUnitTest.java

```diff
@@ -1,253 +1,283 @@
 
         for (int i = numHetGTs; i < numHomRefGTs; i++)
-            allGTs.add(makeG("het" + i, Aref, T, hetPLs));
+            allGTs.add(makeG("het" + i, A_REF, T, HET_PL_ARRAY));
 
-        final VariantContext common = makeVC("common", Arrays.asList(Aref, T), allGTs.toArray(new Genotype[allGTs.size()]));
+        final VariantContext common = makeVC("common", Arrays.asList(A_REF, T), allGTs.toArray(new Genotype[allGTs.size()]));
         final double commonValue = ExcessHet.calculateEH(common, common.getGenotypes()).getValue();
 
         Assert.assertTrue(Math.abs(hundredtonValue) < Math.abs(commonValue), String.format("hundredton=%f common=%f", hundredtonValue, commonValue));
     }
 
     @Test
     public void testAllHetsForLargeCohorts() {
-
         final int numGTs = 1000000;
 
         final List<Genotype> singletonGTs = new ArrayList<>();
         for (int i = 0; i < numGTs; i++) {
-            singletonGTs.add(makeG("ref" + i, Aref, Aref, homRefPLs));
+            singletonGTs.add(makeG("ref" + i, A_REF, A_REF, HOM_REF_PL_ARRAY));
         }
 
-        singletonGTs.add(makeG("het0", Aref, T, hetPLs));
+        singletonGTs.add(makeG("het0", A_REF, T, HET_PL_ARRAY));
 
-        final VariantContext singleton = makeVC("singleton", Arrays.asList(Aref, T), singletonGTs.toArray(new Genotype[singletonGTs.size()]));
+        final VariantContext singleton = makeVC("singleton", Arrays.asList(A_REF, T), singletonGTs.toArray(new Genotype[singletonGTs.size()]));
         final double singletonValue = ExcessHet.calculateEH(singleton, singleton.getGenotypes()).getValue();
 
         final List<Genotype> allHetGTs = new ArrayList<>();
         for (int i = 0; i < numGTs; i++) {
-            allHetGTs.add(makeG("het" + i, Aref, T, hetPLs));
+            allHetGTs.add(makeG("het" + i, A_REF, T, HET_PL_ARRAY));
         }
 
-        final VariantContext allHet = makeVC("allHet", Arrays.asList(Aref, T), allHetGTs.toArray(new Genotype[allHetGTs.size()]));
+        final VariantContext allHet = makeVC("allHet", Arrays.asList(A_REF, T), allHetGTs.toArray(new Genotype[allHetGTs.size()]));
         final double hetsValue = ExcessHet.calculateEH(allHet, allHet.getGenotypes()).getValue();
 
         Assert.assertTrue(Math.abs(singletonValue) < Math.abs(hetsValue), String.format("singleton=%f allHets=%f", singletonValue, hetsValue));
 
         //Since all hets is such an extreme case and the sample size is large here, we know that the p-value should be 0
-        Assert.assertEquals(hetsValue, ExcessHet.PHRED_SCALED_MIN_P_VALUE, DELTA_PRECISION, String.format("P-value of 0 should be phred scaled to " + ExcessHet.PHRED_SCALED_MIN_P_VALUE));
+        Assert.assertEquals(hetsValue, ExcessHet.PHRED_SCALED_MIN_P_VALUE, DELTA_PRECISION, "P-value of 0 should be phred scaled to " + ExcessHet.PHRED_SCALED_MIN_P_VALUE);
     }
 
     @DataProvider(name = "smallSets")
     public Object[][] counts() {
         return new Object[][]{
-                {1, 0, 0, 0.5},
```

Comment:
Consequential test changes start here; most of the other changes in the class are just the result of scout-level cleanup.